code/__DEFINES/dcs/signals/signals_mob/signals_mob_simple.dm

```diff
@@ -10,8 +10,6 @@
 	#define COMPONENT_HOSTILE_NO_ATTACK COMPONENT_CANCEL_ATTACK_CHAIN //cancel the attack, only works before attack happens
 ///after attackingtarget has happened, source is the attacker and target is the attacked, extra argument for if the attackingtarget was successful
 #define COMSIG_HOSTILE_POST_ATTACKINGTARGET "hostile_post_attackingtarget"
-///from base of mob/living/basic/regal_rat: (mob/living/basic/regal_rat/king)
-#define COMSIG_RAT_INTERACT "rat_interaction"
 	#define COMPONENT_RAT_INTERACTED (1<<0) //! If this is returned, cancel any further interactions.
```

Comment:
btw you should move this to where COMSIG_RAT_INTERACT is now

Reply:
Oh! Fixing it asap.